### PR TITLE
MOSIP-28646 code changes (#1757)

### PR DESCRIPTION
**Pull request by sowmya695**

### CODE REVIEW

File: registration-processor/core-processor/registration-processor-abis-handler-stage/src/main/java/io/mosip/registration/processor/abis/handler/stage/AbisHandlerStage.java [hunk 1]
```diff
@@ -593,20 +587,15 @@ private String getDataShareUrl(String id, String process) throws Exception {
...
-					priorityBasedPacketManagerService.getMetaInfo(id, process, ProviderStageName.BIO_DEDUPE));
-		} else {
-			validateBiometricRecord(biometricRecord, modalities, biometricModalitySegmentsMapAdult,
-					priorityBasedPacketManagerService.getMetaInfo(id, process, ProviderStageName.BIO_DEDUPE));
+		Map<String, List<String>> biometricModalitySegmentsMap;
+		if(biometricModalitySegmentsMapforAgeGroup.containsKey(ageGroup)){
+			biometricModalitySegmentsMap = biometricModalitySegmentsMapforAgeGroup.get(ageGroup);
+			}
+		else {
+			biometricModalitySegmentsMap = biometricModalitySegmentsMapforAgeGroup.get("DEFAULT");
```
Comment (vishwa-vyom) [inline registration-processor/core-processor/registration-processor-abis-handler-stage/src/main/java/io/mosip/registration/processor/abis/handler/stage/AbisHandlerStage.java:602]: if the map does not contain default age group, how are we handling that ?

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: registration-processor/core-processor/registration-processor-abis-handler-stage/src/main/java/io/mosip/registration/processor/abis/handler/stage/AbisHandlerStage.java (hunk 1 above)
Comment (sowmya695) [reply registration-processor/core-processor/registration-processor-abis-handler-stage/src/main/java/io/mosip/registration/processor/abis/handler/stage/AbisHandlerStage.java:602]: For Else condition we assumed always DEFAULT should be present  and proceeded without adding any null check.

**Comment by sowmya695** [comment]
closing and reopen another pr to solve sign off,https://github.com/mosip/registration/pull/1845/files
because sign off causing lot of conflicts

File: registration-processor/core-processor/registration-processor-abis-handler-stage/src/main/java/io/mosip/registration/processor/abis/handler/stage/AbisHandlerStage.java (hunk 1 above)
Comment (vishwa-vyom) [reply registration-processor/core-processor/registration-processor-abis-handler-stage/src/main/java/io/mosip/registration/processor/abis/handler/stage/AbisHandlerStage.java:602]: As discussed, @sowmya695 to create a Jira ticket to handle this case.

**Pull request closed without merging**